OpenRA.Mods.RA2/Activities/Infect.cs

```diff
@@ -0,0 +1,162 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Mods.RA2.Traits;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.RA2.Activities
+{
+	class Infect : Enter
+	{
+		readonly AttackInfect attackInfect;
+		readonly Target target;
+
+		bool jousting;
```

Comment:
@Mailaender still `jousting` here

Reply:
Perhaps something like `playingInfectAnim` is more appropriate to this generic (possibly before it needs to be?) trait.